src/cloud-api-adaptor/pkg/securecomms/ppssh/ppssh.go

```diff
@@ -0,0 +1,304 @@
+package ppssh
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"errors"
+	"fmt"
+	"log"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshproxy"
+	"github.com/confidential-containers/cloud-api-adaptor/src/cloud-api-adaptor/pkg/securecomms/sshutil"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	PROVEN_PP_PRIVATE_KEY_PATH  = "/tmp/provenPpPrivateKey"
+	PROVEN_WN_PUBLIC_KEY_PATH   = "/tmp/provenTePublicKey"
+	UNPROVEN_WN_PUBLIC_KEY_PATH = "/tmp/unprovenTePublicKey"
+	SIGNELTON_PATH              = "/tmp/sshSingleton"
```

Comment:
Typo: SINGLETON instead of SIGNELTON

Reply:
This was already fixed in a later commit.